src/api/server/configuration.rst

```diff
@@ -16,316 +16,14 @@
 Configuration
 =============
 
-The CouchDB Server Configuration API provide an interface to query and update
-the various configuration values within a running CouchDB instance.
+The classical CouchDB Server Configuration API located at ``/_config`` will
+return a 404 in CouchDB 2.0 and got replaced by :ref:`HTTP API <api/node>`.
 
-``/_config``
```

Comment:
Separate sections was good solution if we had a plan to keep cluster/node interfaces for a long time. However, we have the other plan to get rid node interface at all, so creating a separate sections that will contains 80% of copy-paste is not good solution, even in short-term.

Special marks made with special sphinx directives is the simplest way to go here.